api/folder.ts

```diff
@@ -16,15 +15,25 @@ export const getFolderNavInfo = async () => {
   }
 };
 
-export const getFolderListInfo = async (id: number) => {
-  let query = "/api/users/1/links";
-  if (id !== navEntireTab) {
-    query = `${query}?folderId=${id}`;
+export const getFolderListInfo = async (
+  navId: string | string[] | undefined,
+  userToken: string | undefined
```

Comment:
undefined 인 경우 아래와 같이 [선택적 프로퍼티](https://poiemaweb.com/typescript-interface#6-%EC%84%A0%ED%83%9D%EC%A0%81-%ED%94%84%EB%A1%9C%ED%8D%BC%ED%8B%B0)로 작성할 수도 있습니다!
```suggestion
  navId?: string | string[],
  userToken?: string
```